Src/MoneyFox.Windows.Tasks/LiveTilesTaskService.cs

```diff
@@ -0,0 +1,1253 @@
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountBalance"),await GetLatestBalanceAsync(accountid)),
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												},
+												new AdaptiveText()
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountLastMonthsExpenses"),await GetMonthExpensesAsync(DateTime.Now.AddMonths(-1).Month,accountid)),
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												},
+											   new AdaptiveText()
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountCurrentMonthsExpenses"), await  GetMonthExpensesAsync(DateTime.Now.Month,accountid)),
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												}
+
+											}
+										}
+									}
+								}
+							}
+								}
+							}
+						}
+					};
+
+					TileNotification tn = new TileNotification(content.GetXml());
+					TileUpdateManager.CreateTileUpdaterForSecondaryTile(accountid.ToString()).Update(tn);
+				}
+			}
+
+		}
+		private async Task<double> GetMonthExpensesAsync(int month, int accountid)
+		{
+			var currentmonth = DateTime.Now.Month;
+			double balance = 0.00;
+			IEnumerable<Payment> payments = await paymentService.GetPaymentsByAccountId(accountid);
+			foreach (Payment item in payments)
+			{
+				if (item.Data.IsRecurring)
+				{
+						GetrecurrancepaymentsMonthly(item, item.Data.RecurringPayment.Recurrence,month);
+				
+				
+				}
+				else
+				{
+					if (item.Data.Date.Month==month && item.Data.Date.Year==DateTime.Now.Year)
+					{
+						allpayment.Add(item.Data.Date, item);
+					}
+				
+				}
+			}
+			foreach (KeyValuePair<DateTime,Payment> item in allpayment)
+			{
+				balance += item.Value.Data.Amount;
+			}
+			return balance;
+		}
+		private async Task<List<string>> Get9PreviouspaymentsAsync()
+		{
+			var acct = await accountService.GetAllAccounts();
+			DateTime today = DateTime.Now;
+			List<Payment> allpayments = new List<Payment>();
+			List<PaymentEntity> newlst = new List<PaymentEntity>();
+			List<RecurringPaymentEntity> allrecurring = new List<RecurringPaymentEntity>();
+			foreach (Account item in acct)
+			{
+				allpayments.AddRange(await paymentService.GetPaymentsByAccountId(item.Data.Id));
+			}
+			foreach (Payment item in allpayments)
+			{
+				if (item.Data.IsRecurring)
+				{
+					Getrecurancepayments(item, item.Data.RecurringPayment.Recurrence, "last");
+				}
+				else
+				{
+					allpayment.Add(item.Data.Date, item);
+				}
+			}
+			var tet = alldates.OrderBy(x => x.Date).TakeLast(8);
+			List<string> returnlist = new List<string>();
+			foreach (var item in tet)
+			{
+				Payment p = allpayment[item];
+				returnlist.Add(string.Format(GetResourceString("LiveTilePastPaymentText"),p.Data.Amount,p.Data.ChargedAccount.Name,p.Data.Date));
+			}
+			alldates.Clear();
+			allpayment.Clear();
+			return returnlist;
+
+		}
+		private async Task<List<string>> Get9NextpaymentsAsync()
```

Comment:
Same here: IMHO method name shouldn't have numbers in it ^^

Reply:
removed numbers